plugins/colorschemes/vscode.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+  helpers.neovim-plugin.mkNeovimPlugin config {
+    name = "vscode";
+    isColorscheme = true;
+    originalName = "vscode-nvim";
+    defaultPackage = pkgs.vimPlugins.vscode-nvim;
+    colorscheme = null; # Color scheme is set by `require.("vscode").load()`
+    callSetup = false;
+
+    maintainers = [maintainers.loicreynier];
+
+    settingsOptions = {
+      transparent = helpers.defaultNullOpts.mkBool false "Whether to enable transparent background";
+      italic_comments = helpers.defaultNullOpts.mkBool false "Whether to enable italic comments";
+      underline_links = helpers.defaultNullOpts.mkBool false "Whether to underline links";
+      disable_nvimtree_bg = helpers.defaultNullOpts.mkBool true "Whether to disable nvim-tree background";
+      color_overrides =
+        helpers.defaultNullOpts.mkStrLuaOr
+        (with helpers.nixvimTypes; attrsOf strLua)
```

Comment:
Shouldn't this be attrsOf anything? From the readme is seems you can do `'#FFFFFF'`. strLua is mean for rawLuaCode.

I think the option should be a defaultNullOpts.mkAttrsOf types.str if I unsterstand the readme.